src/api/spec/features/webui/projects_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "browser_helper"
+
+RSpec.feature "Projects", :type => :feature, :js => true do
+  let!(:user) { create(:confirmed_user) }
```

Comment:
Your VCR cassettes will only work once because each time this will create a different user.